src/rpc/mining.cpp

```diff
@@ -1113,6 +1115,14 @@ static RPCHelpMan estimatesmartfee()
     UniValue errors(UniValue::VARR);
     FeeCalculation feeCalc;
     CFeeRate feeRate = fee_estimator.estimateSmartFee(conf_target, &feeCalc, conservative);
+    CFeeRate min_mempool_feerate = mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000);
+    if (feeRate < min_mempool_feerate) {
+        feeRate = min_mempool_feerate;
+    }
+    CFeeRate min_relay_feerate = ::minRelayTxFee;
+    if (feeRate < min_relay_feerate) {
+        feeRate = min_relay_feerate;
+    }
```

Comment:
Suggestion, you could use braced initialization and `std::max` with an initializer list.
```diff
-    CFeeRate feeRate = fee_estimator.estimateSmartFee(conf_target, &feeCalc, conservative);
-    CFeeRate min_mempool_feerate = mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000);
-    if (feeRate < min_mempool_feerate) {
-        feeRate = min_mempool_feerate;
-    }
-    CFeeRate min_relay_feerate = ::minRelayTxFee;
-    if (feeRate < min_relay_feerate) {
-        feeRate = min_relay_feerate;
-    }
+    CFeeRate feeRate{fee_estimator.estimateSmartFee(conf_target, &feeCalc, conservative)};
+    CFeeRate min_mempool_feerate{mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000)};
+    CFeeRate min_relay_feerate{::minRelayTxFee};
+    feeRate = std::max({feeRate, min_mempool_feerate, min_relay_feerate});
```